interpreter/test/start.wast

```diff
@@ -87,6 +89,11 @@
   (start $main)
 )
 
+(module
+  (func $print (import "spectest" "print"))
+  (start $print)
+)
+
```

Comment:
No, at least on the Web, importing a host function never fails, because they have no types and all JS functions are variadic. The "print" function in "spectest" simulates that behaviour. There already are other tests exercising it.